scheduler/tasks/exporter-task/build.gradle.kts

```diff
@@ -52,11 +53,12 @@ dependencies {
     implementation(JacksonKotlin)
     implementation(JacksonXml)
     implementation(SpringBootStarter)
-    implementation(SpringBootStarterMongo)
+    implementation(SpringBootStarterReactiveMongo)
     implementation(SpringBootStarterWeb)
     implementation(SpringBootStarterConfigProcessor)
 
     testImplementation(KotlinXmlBuilder)
+    testImplementation(KotlinCoroutinesTest)
```

Comment:
In a previous PR I included this as part of the `BaseTestRuntimeDependencies` so I think it no longer needs to be included